modules/installation-and-upgrade/pages/container-deployment/uyuni/migrate-uyuni-server-from-micro55-tumbleweed.adoc

```diff
@@ -0,0 +1,195 @@
+= Migrate {productname} Server from openSUSE Leap Micro 5.5 to Tumbleweed
+:revdate: 2025-10-27
+:page-revdate: {revdate}
+:description: This page describes how to migrate a {productname} Server running on openSUSE Leap Micro 5.5 to a fresh host running openSUSE Tumbleweed as the base OS.
+ifeval::[{mlm-content} == true]
+
+:noindex:
+endif::[]
+
+This page describes a simple, backup-and-restore migration of a {productname} Server running on openSUSE Leap Micro 5.5 to a fresh host running openSUSE Tumbleweed as the base OS.
+
+
+== Overview of the Migration Process
+
+You will:
+
+* Create a full server backup with [command]``mgradm backup`` on the openSUSE Leap Micro 5.5 host.
+* Reinstall the host with openSUSE Tumbleweed (server profile).
+* Install {productname} tools and prerequisites on Tumbleweed.
+* Restore the backup with [command]``mgradm backup restore``.
+* Start services and verify the server.
+
+
+== Requirements and Considerations
+
+* Source server: openSUSE Leap Micro 5.5 running {productname} (for example: {productnumber}).
+* Target server: openSUSE Tumbleweed with the same hostname/FQDN and IP (recommended) to avoid client-side changes.
+* SSH/scp access between machines for transferring the backup tarball.
+* Sufficient free disk space on both source and target for the backup and restore.
+
+[IMPORTANT]
+====
+Restore to the same {productname} version you backed up, or a version explicitly documented as compatible for restore. If you use development or preview repositories (for example, Uyuni Master), expect changes and re-validate.
+====
+
+== Migration Procedure
+
+=== Step 1: Create a Backup on the openSUSE Leap Micro 5.5 Server
+.Procedure:  Create a Backup
+[role="procedure"]
+____
+. As root on the old server, create a backup directory and run the backup:
+
++
+
+[source,shell]
+----
+mgradm backup /tmp/uyuni-backup
+----
+. Package the backup for transfer:
+
++
+
+[source,shell]
+----
+tar -C /tmp -cvf /tmp/uyuni-backup.tar uyuni-backup
+----
+. Copy the backup to a safe location you can reach from the new host:
+
++
+
+[source,shell]
+----
+scp /tmp/uyuni-backup.tar <USER>@<HOST>:/path/to/store/
+----
+
+[TIP]
+====
+You can store the backup to external storage or an object store as long as you can fetch it on the new host.
+====
+
+____
+
+=== Step 2: Reinstall the Host with openSUSE Tumbleweed
+.Procedure: Reinstalling the Host
+[role="procedure"]
+____
+
+. Reprovision the VM or bare-metal host with openSUSE Tumbleweed.
+. Choose a basic “server profile” installation.
+. Set the same hostname/FQDN and IP address as the original server if you want clients to reconnect seamlessly.
```

Comment:
May be add a warning that not doing so would require steps that are not documented here?